integration-tests/src/test/scala/za/co/absa/spline/BasicIntegrationTests.scala

```diff
@@ -91,13 +92,15 @@ class BasicIntegrationTests extends FlatSpec
         }
       })
 
+
+
   "saveAsTable" should "use table path as identifier when writing to external table" in
     withNewSparkSession(spark =>
       withLineageTracking(spark) {
         lineageCaptor => {
           val path = TempDirectory("sparkunit", "table", pathOnly = true).deleteOnExit().path
 
-          spark.sql(s"create table e_table(num int) using parquet location '$path'")
+          spark.sql(s"create table e_table(num int) using parquet location '${path.toUri}'")
```

Comment:
Is there a reason to call `toUri` here and not when creating the path val like on line 74?

Reply:
Yes, `path` is used further in the code (line 111)

Reply:
Ah I see the `toFile`